popup.js

```diff
@@ -0,0 +1,259 @@
+    live_version: 'https://miltonhenschel.github.io/myPortfolio/',
+    source_code: 'https://github.com/miltonHenschel/myPortfolio',
+  }
+];
+
+const body = document.querySelector('body');
+const prjContent = document.querySelector('#portfolio');
+prjContent.innerHTML = `
+            <section class="first-card card">
+                <img class="snapshoot-style" src="images/Snapshoot Portfolio.svg" alt="Snapshoot portfolio image">
+                <section>
+                    <h2 class="card-header font-style on-small">
+                        Tonic
+                    </h2>
+                    <h2 class="on-large card-header font-style">
+                        Facebook 360
+                    </h2>
+                    <section class="card-description">
+                        <ul class="font-style">
+                            <li class="card-list-title on-small"> CANOPY</li>
+                            <li class="on-large card-list-title ">FACEBOOK</li>
+                            <li class="on-large Job-title_and_year">Full Stack Dev</li>
+                            <li class="Job-title_and_year on-small">
+                                Back End Dev
+                            </li>
+                            <li class="Job-title_and_year">
+                                2022
+                            </li>
+                        </ul>
+                    </section>
+                    <p class="notice font-style">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="programming-languages font-style">
+                        <li>html</li>
+                        <li class="on-large">Ruby On Rails</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <button id= "0" class="project-button button" type="button">See Project</button>
+                </section>
+
+            </section>
```

Comment:
- [ ] Good job adding the projects section with JavaScript 💯, but part of the project requirements is implement it dynamically, it is because imagine that you have more than 100 projects, your code will increments a lot. Kindly, iterate the `projects` array and add the title, description, technologies, etc. from it. ⚡It is very similar to what you already did to create the popup windows.